### PR TITLE
Addressing the Scottish Cookbook being silly

### PR DESCRIPTION
**Pull request by Mom-Bun**
#### Summary

SUMMARY: Content "Added a scottish tailoring book, and removing some recipes from the scottish cookbook"

#### Purpose of change

Currently the Scottish Cookbook right now hosts not only cooking recipes, but recipes for making a kilt, bag pipes, and weapons; this seems extremely silly as why would a cookbook, even if its a 13th century book, have detailed instructions on not only being able to make clothing and instruments, but metalworking as well?

#### Describe the solution

I removed all the recipes that made no sense from the Scottish cookbook (Like weapons, the kilt, bag pipes) and added a new tailoring book that hosts the kilt and bag pipe recipes.

Right now the new book doesn't have any recipes besides the bag pipe and kilt, but I do plan in the future to try and add more Scottish clothing that could be added in the tailoring book.

Thanks to @faefux for helping me on this and helping pitch the idea as well.

#### Describe alternatives you've considered

Adjusting the size and weight to make the book much larger, and re-naming it to something that might make more sense; but I had no idea what a good name would be for a book that not only teaches you how to cook, but on tailoring and metalworking.

#### Testing

Ran make style-all-json and everything spat out fine; and I didn't touch any of the C++ code so it should spawn perfectly well.

#### Additional context

N/A

### CODE REVIEW

**Comment by faefux** [comment]
<3

**Comment by I-am-Erk** [comment]
I'm not sure this is all that silly, aside from the fact that cookbooks and recipe books for tailoring and swords didn't exist as we normally think of them in the 13th century. In those days it wouldn't be uncommon to find a scholar's ramblings on various topics related to a central theme, it's more likely than finding a purpose written book.

I think a more appropriate change would be to make it 16th or 17th century and mention that it's a wide ranging and strange rambling about how everything should be done if you want to be a true Scotsman

**Comment by Positronic-Girl** [comment]
Could I probably update the description to make it more modern? The idea of a cookbook also having detailed instructions on how to craft swords and crossbows; and how to make bag pipes just doesn't make sense to me at all to me.

Also on the topic of old books: a cursory glance the Le Viandier is a recipe book from the 13th century and as far as I know; it just has recipes and all that?

File: data/json/items/book/tailor.json [hunk 1]
```diff
@@ -136,5 +136,25 @@
     "max_level": 6,
     "intelligence": 8,
     "time": "30 m"
+  },
+  {
+    "id": "scots_tailor",
+    "type": "BOOK",
+    "name": { "str": "Ye Scots Beuk o Tailoring", "str_pl": "copies of Ye Scots Beuk o Tailoring" },
```
Comment (LaVeyanFiend) [inline data/json/items/book/tailor.json:143]:
```suggestion
    "name": { "str": "Ye Scots Beuk o Tailorin'", "str_pl": "copies of Ye Scots Beuk o Tailorin'" },
```
I refuse to believe the accent would spell it -ing 😛.

**Comment by Night-Pryanik** [comment]
>Could I probably update the description to make it more modern?

Yeah, that would be fine.

**Comment by Positronic-Girl** [comment]
Alright; added suggestions based on what Erk recommended me in the discord and here. I've added the zweihander and kilt to it, putting it at a high level to represent that you need the technical know-how to to figure out how to make it from the scholarly rants, and adjust the description and fun value to represent it being a "silly" thing of being a weird old scottish book while the tailoring book is a more modern variant that is focused on tailoring and thus less fun.

**Pull request merged**